project/gatsby.py

```diff
@@ -0,0 +1,9 @@
+from .remote import RemoteProject
+
+class Gatsby(RemoteProject):
+    upstream_branch = "master"
+    remote = 'https://github.com/gatsbyjs/gatsby-starter-blog.git'
+    updateCommands = {
```

Comment:
```suggestion

    updateCommands = {
```

Also, comment this please.